scripts/gceworker.sh

```diff
@@ -4,7 +4,7 @@ set -euxo pipefail
 
 export CLOUDSDK_CORE_PROJECT=${CLOUDSDK_CORE_PROJECT-${GOOGLE_PROJECT-cockroach-$(id -un)}}
 export CLOUDSDK_COMPUTE_ZONE=${GCEWORKER_ZONE-${CLOUDSDK_COMPUTE_ZONE-us-east1-b}}
-GOVERSION=${GOVERSION-1.7}
+GOVERSION=${GOVERSION-1.7.1}
```

Comment:
It's annoying that A) this requires setting up a new VM from scratch (guess I'll script my customizations this time...) and B) this leaves a (suspended) VM behind that you have to manually delete.